devservices/config.yml

```diff
@@ -58,6 +65,9 @@ x-sentry-service-config:
     taskbroker: [snuba, postgres, relay, taskbroker]
     backend-ci: [snuba, postgres, redis, bigtable, redis-cluster, symbolicator]
     rabbitmq: [postgres, snuba, rabbitmq]
+    symbolicator: [postgres, snuba, symbolicator]
+    local-snuba: [postgres, local-snuba]
+    minimal: [postgres, snuba]
```

Comment:
Running in the local-snuba mode and a mode like minimal would put the user into a bad state since two instances of snuba would be running. We should find a way to avoid this and make modes be able to recognize mutually exclusive modes.

Reply:
removed this, I agree